test/acceptance/tests.py

```diff
@@ -1201,6 +1263,30 @@ def test_feedback_only(self):
         self.staff_area_page.verify_learner_final_score("Final grade: 1 out of 1")
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, MultipeOpenAssessmentMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    @retry()
+    @attr('acceptance')
+    @ddt.data(True, False)
```

Comment:
OK - I will remove that. And that will also help faster run of acceptance tests :P